recipes/freetype/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2.5.2" %}
+
+package:
+    name: freetype
+    version: {{ version }}
+
+source:
+    fn: freetype-{{ version }}.tar.bz2
+    url: http://download.savannah.gnu.org/releases/freetype/freetype-{{ version }}.tar.bz2
+    sha256: 4ff4bd393aa01071ec7b849d035508a505a78f88b2bcf25ff11e58e43c0b9def
+    patches:
+        - subpixel.patch
+        - multi-thread-violations.patch
+        - double-free.patch
+        - CVE-2014-2240.patch
+        - CVE-2014-2241.patch
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - zlib 1.2*
+        - libpng 1.6*
+    run:
+        - zlib 1.2*
+        - libpng 1.6*
+
+test:
+    commands:
+        - freetype-config --version  # [not win]
+        - conda inspect linkages -n _test freetype  # [linux]
+
+about:
+    home: http://freetype.sourceforge.net/index2.html
+    license: GPL-2.0 and FreeType (http://freetype.sourceforge.net/FTL.TXT)
+    summary: A TrueType Font Library
+
+extra:
+    recipe-maintainers:
+        - ccordoba12
```

Comment:
@ccordoba12 I saw you worked on this recipe in conda-recipes, so I added you here too :wink: